modules/core/src/main/scala/ch/epfl/scala/debugadapter/internal/SourceLookUpProvider.scala

```diff
@@ -65,8 +69,8 @@ private[debugadapter] final class SourceLookUpProvider(
     classPathEntries.flatMap(_.fullyQualifiedNames)
   private[internal] def allOrphanClasses: Iterable[ClassFile] =
     classPathEntries.flatMap(_.orphanClassFiles)
-  private[internal] def classesByName(name: String) =
-    classSearch.find(name)
+  private[internal] def classesByName(name: String): Seq[String] =
+    classSearch.find(_._1 == name).map(_._2).getOrElse(Seq.empty)
```

Comment:
`classSearch` is a `Map` so you should do:

```suggestion
    classSearch.get(name).getOrElse(Seq.empty)
```